src/Xamarin.Android.Tools.Bytecode/Tests/Xamarin.Android.Tools.Bytecode-Tests.csproj

```diff
@@ -56,7 +56,13 @@
     <Compile Include="ParameterFixupTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <TestJar Include="java\**\*.java" />
+    <TestJar
+        Include="java\**\*.java"
+        Exclude="java\java\util\Collection.java"
```

Comment:
But then the resulting state is that future test writers are likely lost when they need to add new test files because the build involves two different javac compilation steps? What if they are mutually dependent on the ones in @(TestJar) and @(TestJarNoParameters) ? I still don't think this is the way to go.